Wordle.Api/Wordle.Api/Controllers/WordController.cs

```diff
@@ -35,6 +36,13 @@ public async Task<Word> AddWord(string newWord, bool isCommon)
             return await _wordService.AddWordAsync(newWord, isCommon);
         }
 
+        [HttpDelete("Delete")]
+        public async Task<Word> DeleteWord(string word)
+        {
+            // deletes word
```

Comment:
```suggestion
```
Something Meg said from the prior class - we should be writing code that's readable instead of placing comments everywhere. The code here is quite readable so the comment is a little redundant.

Reply:
lol i just added that to test the pipeline ill remove it